src/Common/RadixSort.h

```diff
@@ -214,6 +215,22 @@ struct RadixSort
     static KeyBits keyToBits(Key x) { return bit_cast<KeyBits>(x); }
     static Key bitsToKey(KeyBits x) { return bit_cast<Key>(x); }
 
+    struct LessComparator
+    {
+        ALWAYS_INLINE bool operator()(Element & lhs, Element & rhs)
+        {
+            return Traits::less(Traits::extractKey(lhs), Traits::extractKey(rhs));
+        }
+    };
+
+    struct GreaterComparator
+    {
+        ALWAYS_INLINE bool operator()(Element & lhs, Element & rhs)
+        {
+            return !Traits::less(Traits::extractKey(lhs), Traits::extractKey(rhs));
```

Comment:
This doesn't seem right. `!less` is greater or equals, not greater.

Reply:
Thanks. Updated.

Reply:
Have you forgotten to push?

Reply:
Yes. Sorry. Updated.